src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  font-family: $font-family;
+  padding: 16px;
+  transition: transform 1s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 0 0;
+  }
+
+  &__header {
+    color: $color-black;
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    margin: 40px 0 0;
+
+  }
+
+  &:hover &__header {
+    color: #34568b;
+  }
+
+  &__product_code {
+    font-weight: 400;
+    font-size: 10px;
+    margin: 4px 0 0;
```

Comment:
Similarly, the margin here should also adhere to the shorthand notation as per the code style rules.